README.md

```diff
@@ -1,114 +1,142 @@
+
+The following table provides a detailed list of all models materialized within this package by default. 
+> TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_linkedin/#!/overview?g_v=1&g_e=seeds).
+
+| **Model**                          | **Description**                                                                                                        |
 | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| [linkedin__ad_adapter](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin__ad_adapter.sql)               | Each record represents the daily ad performance of each creative, including information about the used UTM parameters. |
-| [linkedin__account_ad_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin__account_ad_report.sql)        | Each record represents the daily ad performance of each account.                                                       |
-| [linkedin__campaign_ad_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin__campaign_ad_report.sql)       | Each record represents the daily ad performance of each campaign.                                                      |
-| [linkedin__campaign_group_ad_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin__campaign_group_ad_report.sql) | Each record represents the daily ad performance of each campaign group.                                                |
+| [linkedin_ads__account_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin_ads__account_report.sql)        | Each record represents the daily ad performance of each account.                                                       |
+| [linkedin_ads__campaign_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin_ads__campaign_report.sql)       | Each record represents the daily ad performance of each campaign. Linkedin campaigns map onto ad groups in other ad platforms.                                                      |
+| [linkedin_ads__campaign_group_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin_ads__campaign_group_report.sql) | Each record represents the daily ad performance of each campaign group. Linkedin                                                 |
+| [linkedin_ads__creative_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin_ads__creative_report.sql) | Each record represents the daily ad performance of each creative.                                                |
+| [linkedin_ads__url_report](https://github.com/fivetran/dbt_linkedin/blob/main/models/linkedin_ads__url_report.sql) | Each record represents the daily ad performance of each url.                                                |
```

Comment:
Can we update these links to use the dbt docs links instead? It may be easiest to wait to update these until we have merged to `main` and the docs are officially hosted.

Reply:
will just wanna double check that the links are good after merging and enabling github pages